src/tracker-component.js

```diff
@@ -219,6 +220,7 @@ export const Tracker = (config? : Config = {}) => {
   let userId;
 
   try {
+    new IdentityManager(config);
```

Comment:
why `new IdentityManager`? Usually I use `new` when there are possible multiple instances of the object, but I can't think of a scenario where we would want multiple `IdentityManager`